frontend/src/actions/articleActions.js

```diff
@@ -0,0 +1,35 @@
+const url = 'http://127.0.0.1:8000';
+
+const requestingArticleData = () => ({ type: 'SENDING_ARTICLE_DATA' });
+const receiveResponseArticle = resp => ({
```

Comment:
More a critique. White space is your friend. Don't butt up functions together like this.

Reply:
Oh, probably because you're using a mix of ES5 and ES6 syntax.